app/helpers/remote_execution_helper.rb

```diff
@@ -28,6 +28,8 @@ def job_invocation_chart(invocation)
   def job_invocation_status(invocation)
     if invocation.last_task.blank?
       _('Job not started yet 0%')
+    elsif invocation.last_task.state == 'scheduled'
+      _('Job set to execute at %s') + invocation.last_task.start_at
```

Comment:
I'm getting `can't convert ActiveSupport::TimeWithZone into String` when invoking the job scheduled to future